sdk/communication/azure-communication-chat/tests/_shared/test_utils.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+from datetime import datetime
+import dateutil.tz
```

Comment:
dateutil is not part of the standard Python library - you will need to add this as a dependency in `dev_requirements.txt`.

Reply:
Updated